app/models/grda_warehouse/hud/client.rb

```diff
@@ -728,7 +728,7 @@ def self.extended_age_groups
         range_51_to_55: { name: '51 - 55 yrs old', range: (51..55) },
         range_55_to_60: { name: '56 - 60 yrs old', range: (55..60) },
```

Comment:
```suggestion
        range_55_to_60: { name: '56 - 60 yrs old', range: (56..60) },
```

Reply:
Should `range_44_to_50` be `{ name: '46 - 50 yrs old', range: (46..50) }`? (GH wouldn't let me comment on that line)